conf/README.md

```diff
@@ -0,0 +1,84 @@
+# Config directory
+
+In this directory we will be storing all the configuration for clusters and
+OCSCI config files.
+
+During the execution, via
+[ocsci pytest plugin](../pytest_customization/ocscilib.py), we are loading all
+the configs described below, combining them togehter and using jinja2
+templates rendering variables which you can reuse among different configs from
+DEFAULT and different sections.
+
+This gives us the ability to easily reusing variables from context of the
+yaml file itself.
+
+Cause you can overwrite data just in specific section like for ENV_DATA
+(see below) and if you don't overwrite data in DEFAULT section, someone can
+still access old default value from DEFAULT section directly. See example of
+[this cluster config file](./examples/ocs_basic_install.yml) for more details.
+
+## Cluster config
+
+If you would like to overwrite cluster default data you can create file
+similar to [this example](./examples/ocs_basic_install.yml). You can see we
+can overwrite default data like:
+
+* `platform` - currently we support only AWS, but in future we will cover
+    more.
+* `worker_replicas` - number of replicas of worker.
+* `rook_image` - which rook image can be used.
+* and many more - look at the deployment templates under
+    [../templates/ocs-deployment/] and you can see the variables in templates.
+    In the deploment we are using ENV_DATA section as data for rendering the
+    templates whic allow us to add new values just in config and we don't have
+    to also update the code on all places to take specific variable.
+
+If you define cluster config you have to put all the data under `ENV_DATA`
+section like in the mentioned example. Actually you can add another section as
+well but we recommend int cluster config define just `ENV_DATA` section or one
+more additional section which is `DEFAULTS` for propagation those defaults
+among all configs as well.
+
+## OCS CI Config
```

Comment:
Now I think that it would be better to move this section to the top, making it the 1st one.  This way, you will explain that we have OCS CI config, defaults and sections in our confing. Then it is easier to describe the cluster config (where you talk about the defaults and some sections anyway, but without broader context).

Reply:
moved